lms/static/sass/views/_shoppingcart.scss

```diff
@@ -436,59 +301,66 @@
     }
 
     .data-input {
+      @include float(left);
+      @include margin-left($baseline*1.25);
       width: calc(100% - 245px);
-      float: left;
-      margin-left: 25px;
 
       h3, h3 span {
-        font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-        font-size: 16px;
+        @include line-height(13.51);
+        @extend %t-icon5;
+        padding: 0;
         text-transform: uppercase;
         color: $light-gray2;
-        padding: 0;
-        line-height: 20px;
+        font-family: $sans-serif;
       }
 
       h1 {
-        font-size: 24px;
-        color: $dark-gray1;
+        @include float(left);
+        @extend %t-icon3;
         padding: 0 0 ($baseline/2) 0;
+        width: ($baseline*35);
         text-transform: capitalize;
+        color: $dark-gray1;
+
+        // Adding this content to insure proper display of closing parentheses
+        &:after {
+          content: "\200E‎";
+        }
 
         span {
-          font-size: 16px;
+          @extend %t-icon5;
         }
-        width: 700px;
-        float: left;
       }
 
       span.date {
+        @include float(right);
+        @include text-align(right);
         width: calc(100% - 700px);
```

Comment:
There are couple of these width calculations with hardcoded value. Could these be calculated using baseline and then inserted here?